style.css

```diff
@@ -0,0 +1,204 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  width: 100%;
+  min-height: 100vh;
+}
+
+.container {
+  width: 100%;
+  height: 100vh;
+  color: white;
+}
+
+.container .menu {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 30px;
+  height: 70px;
+}
+
+.container .logo {
+  color: #6070ff;
+  font-style: normal;
+  font-size: 18px;
+  text-decoration: none;
+}
+
+.menu ul {
+  list-style: none;
+}
+
+.menu ul li {
+  display: inline-table;
+  padding: 0 20px;
+}
+
+.menu ul li a {
+  position: relative;
+  display: block;
+  text-decoration: none;
+  color: #fff;
+  font-size: 20px;
+}
+
+.menu ul li a::after {
+  content: '';
+  position: absolute;
+  width: 100%;
+  height: 1px;
+  background-color: yellow;
+  top: 110%;
+  left: 0;
+  border-radius: 5px;
+  transition: 0.4s;
+  transform: scale(0);
+}
+
+.menu ul li a:hover::after {
+  transform: scale(1);
+}
+
+.menu ul li a.active::after {
+  transform: scale(1);
+}
+
+.menu #menuBtn {
+  cursor: pointer;
+  user-select: none;
+  display: none;
+}
+
+#menuBtn .bar {
+  width: 26px;
+  height: 3px;
+  background-color: #fff;
+  margin: 5px auto;
+  border-radius: 5px;
+  transition: 0.4s cubic-bezier(1, 0.2, 0.2, 1);
+}
+
+.container main {
+  width: 100%;
+  height: 100%;
+  background: url(assets/Header-bg.svg) no-repeat center center;
+  background-size: cover;
+  overflow: hidden;
+  display: flex;
+  justify-content: flex-start;
+  align-items: center;
+}
```

Comment:
Please note that yet the background image is not yet as in the FIGMA design

Your implementation | Figma Design
:------:|:------:
![image](https://user-images.githubusercontent.com/83596135/205142938-01878c29-25ea-470a-be84-5ceea0583d01.png) | ![image](https://user-images.githubusercontent.com/83596135/205143015-874c04b5-518e-4e25-b41f-541309b81f78.png)

- [ ] Kindly take the necessary measures to have the background image properly displayed.